### PR TITLE
[ix]: minor chore updates

### PR DESCRIPTION
**Pull request by yash251**

### CODE REVIEW

File: website/pages/en/about.mdx [hunk 1]
```diff
@@ -44,4 +44,4 @@ The flow follows these steps:
 
 In the following sections we will go into more detail on how to define subgraphs, how to deploy them, and how to query data from the indexes that Graph Node builds.
 
-Before you start writing your own subgraph, you might want to have a look at the Graph Explorer and explore some of the subgraphs that have already been deployed. The page for each subgraph contains a playground that lets you query that subgraph's data with GraphQL.
+Before you start writing your own subgraph, you might want to have a look at the [Graph Explorer](https://thegraph.com/explorer) and explore some of the subgraphs that have already been deployed. The page for each subgraph contains a playground that lets you query that subgraph's data with GraphQL.
```
Comment (benface) [inline website/pages/en/about.mdx:47]:
```suggestion
Before you start writing your own subgraph, you might want to have a look at [Graph Explorer](https://thegraph.com/explorer) and explore some of the subgraphs that have already been deployed. The page for each subgraph contains a playground that lets you query that subgraph's data with GraphQL.
```

Source: https://www.notion.so/edgeandnode/The-Graph-Style-Guide-70a1409024d84bfbb4eccdf0e49b7aa7?pvs=4#4baa7d6f012a43bbadfc1d7180d9c47e (thanks @bermchain!)

File: website/pages/en/network/indexing.mdx [hunk 2]
```diff
@@ -182,7 +182,7 @@ Note: To support agile scaling, it is recommended that query and indexing concer
 
 #### Create a Google Cloud Project
 
-- Clone or navigate to the Indexer repository.
+- Clone or navigate to the [Indexer repository](https://github.com/graphprotocol/indexer).
 
 - Navigate to the ./terraform directory, this is where all commands should be executed.
```
Comment (benface) [inline website/pages/en/network/indexing.mdx:187]:
```suggestion
- Navigate to the `./terraform` directory, this is where all commands should be executed.
```

I know you didn't change this, but I saw it right below a line you changed so why not.

File: website/pages/en/sunrise.mdx [hunk 3]
```diff
@@ -38,15 +38,15 @@ To upgrade a hosted service subgraph, you can visit the subgraph dashboard on th
 2. Select the receiving wallet (the wallet that will become the owner of the subgraph).
 3. Click the "Upgrade" button.
 
-Once your subgraph is published, the [upgrade Indexer](#what-is-an-upgrade-indexer) will begin serving queries on it, so you can begin making queries immediately, once you have generated an API key ([learn more](/cookbook/upgrading-a-subgraph/#what-next)).
+Once your subgraph is published, the [Upgrade Indexer](#what-is-the-upgrade-indexer) will begin serving queries on it, so you can begin making queries immediately, once you have generated an API key ([learn more](/cookbook/upgrading-a-subgraph/#what-next)).
```
Comment (benface) [inline website/pages/en/sunrise.mdx:41]: No, "upgrade" [shouldn't be capitalized](https://www.notion.so/edgeandnode/The-Graph-Style-Guide-70a1409024d84bfbb4eccdf0e49b7aa7?pvs=4#09d4ed7a5a5d423aa1074e7ff58e53c1):
```suggestion
Once your subgraph is published, the [upgrade Indexer](#what-is-the-upgrade-indexer) will begin serving queries on it, so you can begin making queries immediately, once you have generated an API key ([learn more](/cookbook/upgrading-a-subgraph/#what-next)).
```

**Review by benface — CHANGES_REQUESTED** [review]
Looks like the lint is failing, can you please run `pnpm check:fix`? 🙏

**Review by benface — APPROVED** [review]

**Review by azf20 — APPROVED** [review]

**Pull request merged**